### ISSUE TITLE
Hardened cache causing non-full blocks to become translucent.

### ISSUE THREAD

**Issue by epilimic** [CoFH/Feedback#976]
Hello, I'm having an issue with the hardened cache. Whenever there's one placed in the world it will cause certain blocks to become translucent and weird looking.

Here's an example: https://i.imgur.com/71IPu1E.png

This also happens to chests, and my bibliocraft typesetting table and printing press.

Here's my mod list: http://i.imgur.com/6Zdtlnq.png

This does not cause a crash so I do not have a log to attach, nor are there any errors logged when this happens.

**Comment by KingLemming** [comment]
Yeah I can't make this happen. It seems x-mod. Caches properly reset all of the render flags that they set.

However, it's possible that another block is expecting certain flags to NOT be reset, and the order that Minecraft is choosing to render things in, based on relative player position, hardware, and whatnot, causes this.

I'll keep it open, but as it stands I can't even get it to happen.

**Comment by skyboy** [comment]
It's also possible the item in the cache is causing it. Those are fun.

**Comment by epilimic** [comment]
It was a cache of blaze rods.
